src/main/java/jsr223/perl/utils/PerlPropertyLoader.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * ProActive Parallel Suite(TM):
+ * The Open Source library for parallel and distributed
+ * Workflows & Scheduling, Orchestration, Cloud Automation
+ * and Big Data Analysis on Enterprise Grids & Clouds.
+ *
+ * Copyright (c) 2007 - 2017 ActiveEon
+ * Contact: contact@activeeon.com
+ *
+ * This library is free software: you can redistribute it and/or
+ * modify it under the terms of the GNU Affero General Public License
+ * as published by the Free Software Foundation: version 3 of
+ * the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ * If needed, contact us to obtain a release under GPL Version 2 or 3
+ * or a different license than the AGPL.
+ */
+package jsr223.perl.utils;
+
+import java.io.IOException;
+import java.util.Properties;
+
+import lombok.Getter;
+import lombok.extern.log4j.Log4j;
+
+
+@Log4j
+public class PerlPropertyLoader {
+
+    private final static String CONFIGURATION_FILE = "config/scriptengines/perl.properties";
```

Comment:
Add this file to the scheduler project (If you really need it). In this file you can setup the perl command, if perl uses sudo, annd all the Properties you use in this code.